app/models/user.rb

```diff
@@ -342,8 +342,6 @@ def self.to_csv
   validates_attachment :picture, content_type: { content_type: ["image/jpeg", "image/gif", "image/png", "image/jpg"] }, unless: -> { self.first_step }
 
   validates_presence_of :name
-  validates_presence_of :birthday, unless: -> { self.is_admin }
```

Comment:
Are there any impact on the app by removing this requirement? I mean, is there any code that expects this to be set?